kubernetes/kserve/tests/scripts/test_mnist.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
```

Comment:
a bit conflicted about this, ideally all our tests should run from regression or sanity tests that way people have a clear way of reproducing failures both locally and remotely 

Do we need the extra json and yaml files? Can't we leverage more of the existing kserve tests?

Reply:
Re-used the json files we have. We don't have the yaml files.